media-tv/kodi/kodi-9999.ebuild

```diff
@@ -234,28 +234,27 @@ src_configure() {
 		$(use_enable X x11)
 }
 
-src_compile() {
-	emake V=1
```

Comment:
Indeed, the `--disable-silent-rules` from autoconf is useless because upstream uses their own `Makefile.in` instead of automake. So yes, @candrews you'll have to switch back to the old syntax 😭.